otx/api/entities/tensor.py

```diff
@@ -21,7 +21,7 @@ class TensorEntity(IMetadata):
 
     def __init__(self, name: str, numpy: np.ndarray):
         self.name = name
-        self._numpy = numpy
+        self._numpy = np.copy(numpy)
```

Comment:
Could you explain why we need copy of numpy?

Reply:
It's reflecting the change in develop branch to feature/otx: 
https://github.com/openvinotoolkit/training_extensions/pull/1511

I didn't create a new PR as @goodsong81 suggested I add here